src/agentlab/agents/dynamic_prompting.py

```diff
@@ -245,9 +245,7 @@ def fit_tokens(
         additional_prompts = [additional_prompts]
 
     for prompt in additional_prompts:
-        max_prompt_tokens -= (
-            count_tokens(prompt, model=model_name) + 1
-        )  # +1 accounts for LangChain token
+        max_prompt_tokens -= count_tokens(prompt, model=model_name) + 1  # +1 because why not ?
```

Comment:
lol

Reply:
actually it's more about being conservative on the token count, i'd rather cut 5 tokens short than be 5 tokens too long and have an error